site/dat/docs/Appium.md

```diff
@@ -0,0 +1,26 @@
+# Appium Executor
```

Comment:
Appium is not executor, it happened to be service. So we need to explain that this is service module for Selenium executor type.